### PR TITLE
Add metrics endpoint debugging info for container docs

### PR DESCRIPTION
**Pull request by bmorelli25**
### Summary

This PR adds metrics endpoint debugging information.

Closes https://github.com/elastic/observability-docs/issues/891.

### CODE REVIEW

**Comment by bmorelli25** [comment]
@michalpristas what I have right now is pretty basic and based on limited notes. If you think it makes sense to meet and chat about this before moving forward, let me know.

File: docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc [hunk 1]
```diff
@@ -137,3 +131,224 @@ Each subprocess writes its own logs to the `default` directory inside the logs d
...
+  metrics: true <3>
+  http:
+      enabled: true <4>
+      host: localhost <5>
+      port: 6791 <6>
+----
+<1> Enable monitoring of running processes.
+<2> Enable log monitoring.
+<3> Enable metrics monitoring.
+<4> Enable {agent} metrics using HTTP. By default, sockets and named pipes are used.
```
Comment (michalpristas) [inline docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc:159]: `exposing metrics` maybe?  just to be more precise/clear. collecting metrics is enabled by `1-3`

**Review by michalpristas — APPROVED** [review]
lgtm thanks for this

File: docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc (hunk 1 above)
Comment (bmorelli25) [reply docs/en/ingest-management/elastic-agent/elastic-agent-container.asciidoc:159]:
```suggestion
<4> Expose {agent} metrics over HTTP. By default, sockets and named pipes are used.
```

**Pull request merged**